### PR TITLE
Add new py-pydv package

### PR DESCRIPTION
**Pull request by DarylGrunau**
This PR adds support to build/install PDV, a 1D graphics and data analysis tool, heavily based upon the ULTRA plotting utility.  NOTE: if your blas/lapack provider is intel-mkl see issue #11289.

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-pydv/package.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+# ----------------------------------------------------------------------------
```
Comment (erimar77) [inline var/spack/repos/builtin/packages/py-pydv/package.py]: Still contains some of the boilerplate, can you remove these lines?

File: var/spack/repos/builtin/packages/py-pydv/package.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+
+class PyPydv(PythonPackage):
+    """PDV is a 1D graphics and data analysis tool, heavily based on the
+    ULTRA plotting tool"""
+
+    homepage = "https://github.com/griffin28/PyDV"
+    url      = "https://github.com/griffin28/PyDV.git"
+
+    version('2.4.2', git=url, tag='pydv-2.4.2')
```
Comment (erimar77) [inline var/spack/repos/builtin/packages/py-pydv/package.py]: Is there a reason you're not using the tarball from the releases page?

Comment (DarylGrunau) [reply var/spack/repos/builtin/packages/py-pydv/package.py]: Whoops, didn't realize that was there; retooling ...

Comment (DarylGrunau) [reply var/spack/repos/builtin/packages/py-pydv/package.py]: Pushed a change to use the tarball from the releases page; all CI checks passed - resolving

**Pull request merged**